terminatorlib/terminal.py

```diff
@@ -972,6 +972,8 @@ def on_buttonpress(self, widget, event):
 
     def on_mousewheel(self, widget, event):
         """Handler for modifier + mouse wheel scroll events"""
+        if self.config["disable_mousewheel_zoom"] is True:
+            return (True)
```

Comment:
This handler is common for all scrolling actions. This therefore also disables scrolling. This should be checked a few lines below where modifiers are checked.

Reply:
Ah yes, I definitely forgot to think here.